samples/charts/category-chart/tooltip-types/Pages/CategoryChartTooltipTypes.razor

```diff
@@ -0,0 +1,85 @@
+﻿@page "/CategoryChartTooltipTypes"
+
+@using Infragistics.Blazor.Controls
+@inject IInfragisticsBlazor InfragisticsBlazor;
+@inject IJSRuntime JSRuntime;
+
+<div class="igContainer">
+    <div class="igOptions">
+        <span className="igOptions-label">Tooltip Type: </span>
+        <select @onchange="OnTooltipTypeChange">
+            <option>Default</option>
+            <option>Item</option>
+            <option>Category</option>            
+            <option>None</option>
```

Comment:
perpend all values of options with "ToolTipType." this way, it will simply `OnTooltipTypeChange()`